pkg/defaults/defaults.go

```diff
@@ -294,7 +294,7 @@ const (
 	// ForceLocalPolicyEvalAtSource is the default value for
 	// option.ForceLocalPolicyEvalAtSource. It is enabled by default to
```

Comment:
Could you update also this comment please to reflect the new default?

Reply:
Sure I will update the comment as well.